cishouseholds/derive.py

```diff
@@ -29,6 +30,25 @@
 from cishouseholds.pyspark_utils import get_or_create_spark_session
 
 
+def assign_window_status(
+    df: DataFrame,
+    column_name_to_assign: str,
+    window_start_column: str,
+    window_end_column: str,
+    current_date: datetime = datetime.now(),
+):
+    """
+    Derive the status of the survey window for a given point in time
+    """
+    df = df.withColumn(
+        column_name_to_assign,
+        F.when((F.col(window_start_column) < current_date) & (current_date <= F.col(window_end_column)), "Open")
```

Comment:
based on the ticket needs to be <= to for first condition:
window_start_column <= current_date